lib/sendgrid/mailer.ex

```diff
@@ -58,7 +57,9 @@ defmodule SendGrid.Mailer do
       reply_to: email.reply_to,
       send_at: email.send_at,
       template_id: email.template_id,
-      mail_settings: %{ sandbox_mode: %{ enable: @sandbox_mode } }
+      mail_settings: %{ sandbox_mode: %{ enable: sandbox_mode } }
     }
   end
+
+  defp sandbox_mode, do: Application.get_env(:sendgrid, :sandbox_enable) || false
```

Comment:
I'm not sure I see the use-case for making the call to get the value every time as opposed to leaving it as a module constant. I don't feel like the value should be modified during runtime.